lib/travis-encrypt.js

```diff
@@ -28,19 +31,39 @@ function getRepoNotFoundError (options) {
   return new Error('repository ' + options.owner + '/' + options.repo + ' not found');
 }
 
-function encryptTravisProData (options, callback) {
+function encryptTravisProData (options, callback, type) {
```

Comment:
Please move `type` to second argument to match callback style always being the last argument